### ISSUE TITLE
[False positive] ::class result and the class qualified name are not identical when aliasing class

### ISSUE THREAD

**Issue by Koc** [kalessil/phpinspectionsea#953]
```php
use Ololo\FrontEndBundle\Entity\Product\Product as ProductSearch;

// displays error
ProductSearch::class;
```

**Comment by kalessil** [comment]
Oh, how did I miss that case... Thank you for reporting @Koc

**Comment by kalessil** [comment]
@Koc, cannot reproduce. I suspect that NS in the `Ololo\FrontEndBundle\Entity\Product\Product` class is not identical to `Ololo\FrontEndBundle\Entity\Product`. Would you please check if this is a case?

**Comment by Koc** [comment]
Hm, this is strange. PHP autoloading loads this aliased class without any error on Linux (with case sensetive filesystem)

**Comment by kalessil** [comment]
Em, so was it a case with case mismatch?

**Comment by kalessil** [comment]
ping @Koc

**Comment by Koc** [comment]
Will recheck on Monday (because haven't access to code until it).

**Comment by Koc** [comment]
Ha!

This class was imported twice: one time without alias and one more time with alias. When I've removed first import - error has gone.

Not sure, can double import create new inspection?)

**Comment by kalessil** [comment]
Make sense, can you provide a specific example as I'm not sure for 100% how the issues is looking like practically.

**Comment by Koc** [comment]
```php
use Ololo\FrontEndBundle\Entity\Product\Product;
use Ololo\FrontEndBundle\Entity\Product\Product as ProductSearch;

ProductSearch::class;
```

**Comment by kalessil** [comment]
For myself: scan all imports.

**Comment by mattsches** [comment]
Wonder if this is related or if I should open a new issue, but using EA Ultimate v2.0.8 I get the same warning when I alias a class like this:

```php
<?php

use App\Http\Requests\FeedRequest as StoreRequest;

doSomething(StoreRequest::class);
```

![auswahl_038](https://user-images.githubusercontent.com/68414/46938340-cd546500-d063-11e8-85d8-5c1da94c181d.png)

**Comment by kalessil** [comment]
Fixed finally!
